packages/lumx-react/src/components/badge/Badge.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import { mdiHeart } from '@lumx/icons';
+import { AspectRatio, Icon, Size, Thumbnail, ThumbnailVariant } from '@lumx/react';
+import { decorators } from '@lumx/react/story-block';
+import { text } from '@storybook/addon-knobs';
+import React from 'react';
+import { ColorPalette } from '..';
+import { Badge } from './Badge';
```

Comment:
Badge and ColorPalette should be imported from '@lumx/react'
Please consider sorting imports